### PR TITLE
team slug is citext now, restore previous admins for premium teams 

### PR DESCRIPTION
**Pull request by seuros**
!502 and !503

### CODE REVIEW

File: db/migrate/20150110084027_change_team_slug_to_citext.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeTeamSlugToCitext < ActiveRecord::Migration
+  def change
+    change_column :teams, :slug, :citext, null: false, index: true , unique: true
```
Comment (rohitpaulk) [inline db/migrate/20150110084027_change_team_slug_to_citext.rb]: Does this create an index? It doesn't seem to appear in `schema.rb`.

Comment (seuros) [inline db/migrate/20150110084027_change_team_slug_to_citext.rb]: normally, yes. We are in rails 3.2.

Comment (rohitpaulk) [inline db/migrate/20150110084027_change_team_slug_to_citext.rb]: Doesn't seem to happen on my local. The column gets changed, but the index isn't added. Could you verify whether the index has been added in production? 

Also, a small note: The `schema.rb` was not updated after you changed the index to unique. No big deal, it should get rewritten whenever the next migration is written :)

Comment (seuros) [inline db/migrate/20150110084027_change_team_slug_to_citext.rb]: the schema.rb won't change. I did not add another migration. 
Schema.rb contain the timestamp of last migration .

Comment (rohitpaulk) [inline db/migrate/20150110084027_change_team_slug_to_citext.rb]: The schema.rb not changing is okay, as long as the index does get added. I ran this same migration on my local and checked whether the index is present via `psql`, but it doesn't show. The only index present is `"teams_pkey" PRIMARY KEY, btree (id)`

**Pull request merged**